frontend/WikiContrib-Frontend/src/App.css

```diff
@@ -656,14 +656,16 @@ h3.accounts  {
 
 .ui.card.commits_load {
   width: 100% !important;
-  padding: 2% !important;
+  padding: 0.5% !important;
   box-shadow: 0 4px 16px 0 rgba(0, 0, 0, 0.25) !important;
+  margin-bottom: -0.6rem;
```

Comment:
You can remove this margin; with this the space between the activity boxes have reduced quite a bit.